src/Raven.Studio/typescript/models/auth/certificateModel.ts

```diff
@@ -89,9 +90,11 @@ class certificateModel {
             case "ClusterNode":
                 return ["All"];
             default:
-                const access = Object.keys(certificateDefinition.Permissions);
-                if (access.length) {
-                    return _.sortBy(access, x => x.toLowerCase());
+                const allowedDatabses = Object.keys(certificateDefinition.Permissions);
```

Comment:
I wonder if we should change return type to dict instead of concat strings?

Reply:
not sure if this is easy to do.

Reply:
done